include/wpe/view-backend.h

```diff
@@ -93,14 +93,19 @@ WPE_EXPORT
 int
 wpe_view_backend_get_renderer_host_fd(struct wpe_view_backend*);
 
+enum wpe_view_state {
```

Comment:
Can this, along with the getter, setter and the variable name, be called e.g. wpe_view_activity_state, to keep it more descriptive and less generic?

Reply:
I did this on purpose, I've always found the wk naming confusing with "activity" there.